src/packages/relations/relations/types.ts

```diff
@@ -17,3 +22,8 @@ export interface UmbRelationDetailModel {
 	createDate: string;
 	comment: string | null;
 }
+
+export type UmbReferenceModel =
```

Comment:
I think we should get our types for these, but it might be something we do after this release.